### PR TITLE
Soobbb/solo project

### PR DESCRIPTION
**Pull request by soobbb**

### CODE REVIEW

File: src/components/Item.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+}) {
+  return (
+    <li className="cart-item-body">
+      <input
+        type="checkbox"
+        className="cart-item-checkbox"
+        onChange={(e) => {
+          handleCheckChange(e.target.checked, item.id)
+        }}
+        checked={checkedItems.includes(item.id) ? true : false} >
```
Comment (plla2) [inline src/components/Item.js:15]: 오... 함수만들어서 잘쓰셨네요

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+function App() {
+  return (
+    <Router>
+      <Nav />
+      <Routes>
+        <Route path="/" element={<ItemList />} />
+        <Route path="/1" element={<ProductList />} />
+        <Route path="/2" element={<BookmarkList />} />
+      </Routes>
```
Comment (kimploo) [inline src/App.js:18]: 유어클래스 요구사항을 잘 읽어보시고, path를 수정해주세요.
https://urclass.codestates.com/content/a2f0a3f5-8b46-490e-915b-4768c357f740?playlist=4826

1, 2, 3이 아니고, 의미가 있는 URI path를 사용하기를 권합니다.

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
```
Comment (kimploo) [inline src/components/Footer.js:1]: React default import 부분은 생략할 수 있습니다. 
https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform

File: src/components/Item.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
```
Comment (kimploo) [inline src/components/Item.js:1]: React default import 부분은 생략할 수 있습니다.
https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform

File: src/components/Nav.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+  const toggleDropdown = () => {
+    setIsOpen(!isOpen);
+  };
+
+  return (
+    <nav id="nav-body">
+      <span id="title">
+        <img className={`${styles.logo}`} src="로고.png" alt="로고" />
+        <a className={`${styles.logo_name}`}>COZ Shopping</a>
```
Comment (kimploo) [inline src/components/Nav.js:16]: Anchor 태그가 필요하다면 정확한 href 값을 넣어주세요.

> The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/HEAD/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
```
Comment (kimploo) [inline src/index.js:1]: React default import 부분은 생략할 수 있습니다.
[https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform)

File: src/pages/BookmarkList.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (kimploo) [inline src/pages/BookmarkList.js:1]: React default import 부분은 생략할 수 있습니다.
[https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform)

File: src/pages/ItemList.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (kimploo) [inline src/pages/ItemList.js:1]: React default import 부분은 생략할 수 있습니다.
[https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform)

File: src/pages/ProductList.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (kimploo) [inline src/pages/ProductList.js:1]: React default import 부분은 생략할 수 있습니다.
[https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#whats-different-in-the-new-transform)

**Pull request merged**